src/entities/person/possible-duplicates.ts

```diff
@@ -0,0 +1,82 @@
+import { Universe } from 'src/universe'
+import { EntityRawPayload, UniverseEntityOptions } from '../_base'
+import { PersonRawPayload, PersonPhonenumberRawPayload } from './person'
+import { ChannelUserRawPayload } from './channel-user'
+import { BaseError } from '../../errors'
+
+export interface PossibleDuplicatesRawPayload extends EntityRawPayload {
+  readonly person_id: string
+  readonly strategies: PossibleDuplicatesStrategies
+}
```

Comment:
Likely this interface is not aligned with the return data

Reply:
It does actually, the return data in the console in FE (see screenshot) matches with what I'd also get back when using postman to access BE directly, or do I might get you wrong here? then please be so kind to specify :) 

The returned data is currently of form `{person_id: uuid, strategies: { global_phonenumber[], exact_name_matches[] }}` and when we add new strategies like `exact_email_matches` etc. then we can add it as a new property of type `StrategiesResult` to `PossibleDuplicatesStrategies` and it will have `document` and `kind` as properties like the others. Maybe `StrategiesDocument` needs to be extended then.

<img width="967" alt="Screenshot 2021-09-13 at 20 06 48" src="https://user-images.githubusercontent.com/9516580/133134435-3d1bea93-be7a-4dad-acd9-508e6739740e.png">

Reply:
I think I might get now what you were hinting or questioning? in the previous version the checkDuplicates was wrongly assuming it would return `PossibleDuplicatesRawPayload[]` which the API isn't! in the return data (`data[0]`) is instead only the object `PossibleDuplicatesRawPayload` that has a property `strategies` which again has the list with the duplicated contacts!

later when we want to return for all people each their duplicates then we actually would have `PossibleDuplicatesRawPayload[]` but for now this was about checking duplicates for one person. WDYT?